ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/views/PaywallFooterView.kt

```diff
@@ -73,6 +73,7 @@ open class PaywallFooterView : FrameLayout {
         override fun onRestoreStarted() { listener?.onRestoreStarted() }
         override fun onRestoreCompleted(customerInfo: CustomerInfo) { listener?.onRestoreCompleted(customerInfo) }
         override fun onRestoreError(error: PurchasesError) { listener?.onRestoreError(error) }
+        override fun onPurchaseCancelled() { listener?.onPurchaseCancelled() }
```

Comment:
Total nitpick but I would move it up to be with the other `onPurchase...` callbacks